src/ContentType.php

```diff
@@ -187,7 +187,7 @@ private function detectFromExtension(ServerRequestInterface $request): ?string
     private function detectFromHeader(ServerRequestInterface $request): ?string
     {
         $headers = call_user_func_array('array_merge', array_column($this->formats, 'mime-type'));
-        $accept = $request->getHeaderLine('Accept');
+        $accept = $request->getHeaderLine('Accept')?: '*/*';
```

Comment:
Mmm, I was thinking in something more "strict" such:

```php
        if (!$request->hasHeader('Accept') {
            return $this->defaultFormat;
        }
```

There may be cases in which the header exists but is empty or evaluated as false (for example, with `"0"`).